apps/settings/views.py

```diff
@@ -90,9 +91,31 @@ def integrations(request):
     return render(request, "settings/integrations.html", context=data)
 
 
+@csrf_exempt
 def data_portability(request):
     """ Export bookmarks """
-    return render(request, "settings/dataportability.html")
+    from apps.web.models import Bookmarks
+    import csv
+
+    user = request.user
+    bookmarks = Bookmarks.objects.filter(user=user)\
+        .values('title', 'link', 'description')
+    data = {}
+    if bookmarks:
+        data = {"bookmarks": list(bookmarks)}
+    if request.method == "POST":
+        if bookmarks:
```

Comment:
If there aren't any bookmarks, maybe we should return a message saying something like: "No Bookmarks found, nothing to export"